.gitlab-ci.yml

```diff
@@ -141,6 +141,7 @@ docs:
         -p ink_primitives -p ink_prelude
         -p ink_lang -p ink_lang_macro -p ink_lang_ir -p ink_lang_codegen
     - mv ${CARGO_TARGET_DIR}/doc ./crate-docs
+    - cargo test --doc --workspace
```

Comment:
We should put this under the CI testing section.

Reply:
Updated the PR.